site/index.rst

```diff
@@ -135,7 +135,7 @@ Course Notes
     topics/fileIO/fileIO
     topics/exceptions/exceptions
     topics/objects/introduction
-    topics/objects/methods
+    topics/objects/interacting
```

Comment:
I do kinda think methods fits better. /shrug

Reply:
🤔

Reply:
Given that "Methods" will probably be more helpful when doing a quick look at the table of contents, I agree --- Methods is perhaps better. Sections can be more precise about interacting objects.